modules/library/src/main/scala/zio/elasticsearch/aggregation/Aggregations.scala

```diff
@@ -210,6 +210,48 @@ private[elasticsearch] final case class Multiple(aggregations: Chunk[SingleElast
     aggregations.map(_.toJson).reduce(_ merge _)
 }
 
+sealed trait PercentilesAggregation
+    extends SingleElasticAggregation
+    with HasMissing[PercentilesAggregation]
+    with WithAgg {
+
+  /**
+   * Sets the `percents` parameter for the [[zio.elasticsearch.aggregation.PercentilesAggregation]].
+   *
+   * @param percents
+   *   a array of percentiles to be calculated for [[zio.elasticsearch.aggregation.PercentilesAggregation]]
+   * @return
+   *   an instance of the [[zio.elasticsearch.aggregation.PercentilesAggregation]] enriched with the `percents`
+   *   parameter.
+   */
+  def percents(percent: Double, percents: Double*): PercentilesAggregation
+}
+
+private[elasticsearch] final case class Percentiles(
+  name: String,
+  field: String,
+  percents: Chunk[Double],
```

Comment:
Let's put `missing` before `percents`. I prefer alphabetically sorted fields and methods.